README.md

```diff
@@ -1 +1,70 @@
-# bookstore
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [OPTIONAL] Please provide project title, description, screenshots, live demo, installations and setup section, information about the author, issues page link, and license in your Readme.